make.config.in

```diff
@@ -172,6 +173,9 @@ install: libfast
 	$(INSTALL_DATA)  tools/pylib/boutdata/*.py $(DESTDIR)@datadir@/bout++/pylib/boutdata/
 	$(INSTALL_DATA)  tools/pylib/boututils/*.py $(DESTDIR)@datadir@/bout++/pylib/boututils/
 	$(INSTALL_DATA)  make.config $(DESTDIR)@datadir@/bout++/
+	$(INSTALL_DATA)  locale/fr/LC_MESSAGES/libbout.mo $(DESTDIR)@localedir@/fr/LC_MESSAGES/
+	$(INSTALL_DATA)  locale/zh_TW/LC_MESSAGES/libbout.mo $(DESTDIR)@localedir@/zh_TW/LC_MESSAGES/
+	$(INSTALL_DATA)  locale/zh_CN/LC_MESSAGES/libbout.mo $(DESTDIR)@localedir@/zh_CN/LC_MESSAGES/
```

Comment:
Is there a way to install all available languages, rather than manually listing them here?

Reply:
Hopefully; my make-fu failed as I couldn't think of a nice way to do this loop. The list of .mo files can be got quite easily, but turning this into an install target is a bit more tricky. 
I'm not sure that locale works currently with `install`; the location of the .mo files may not be correctly set.